test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -673,7 +673,7 @@
           (mt/with-report-timezone-id! "UTC"
             (is (= "2022-10-03T14:10:20+07:00"
                    (->> (mt/run-mbql-query times
-                          {:expressions {"expr" [:convert-timezone "2022-10-03T07:10:20" "Asia/Saigon" "UTC"]}
+                          {:expressions {"expr" [:convert-timezone "2022-10-03T07:10:20" "Asia/Ho_Chi_Minh" "UTC"]}
```

Comment:
`Asia/Saigon` seems to be obsolete and was replaced by
`Asia/Ho_Chi_Minh`.

Not entirely sure if this is necessary because the `mariadb:latest`
image has new tzinfo in the container itself, or if the MariaDB docker
image itself has newer tzinfo, but - seems safe enough to replace the
old time zone with the one that it links to.

See https://en.wikipedia.org/wiki/List_of_tz_database_time_zones